apps/site/assets/ts/schedule/components/schedule-finder/ScheduleTable.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactElement } from "react";
+import { ServiceScheduleInfo } from "../__schedule";
+import TableRow from "./TableRow";
+
+const ScheduleTable = ({
+  schedule
+}: {
+  schedule: ServiceScheduleInfo;
+}): ReactElement<HTMLElement> => {
+  if (schedule.trip_order.length === 0) {
+    return (
+      <div className="callout schedule-table--empty">
+        There are no scheduled departures for this service
```

Comment:
@ingridpierre LMK if you want to change this language for when there is no service (but a service category exists). I took it from the current Schedule tab.

Reply:
@meagonqz  How about: "There is no scheduled service for this time period." 

Alternatively, I'd love to say "There is no scheduled service on [date]" or "There is no scheduled service between [date range]". Just to help connect the dots for folks if they happen to visit the page on a holiday and it defaults to a "no service" message.